flink-ml-iteration/src/main/java/org/apache/flink/iteration/operator/OperatorUtils.java

```diff
@@ -104,6 +110,50 @@ public static StreamConfig createWrappedOperatorConfig(
             }
         }
 
+        StreamConfig.InputConfig[] inputs = wrapperConfig.getInputs(cl);
```

Comment:
It's a good idea to rename `wrapperConfig` to `config`, but I don't think it matters to use `wrapperConfig` or `wrappedConfig`. Because the `wrappedConfig` is just a clone from `wrapperConfig` and we are getting the new configurations from the `wrapperConfig` and set to the `wrappedConfig`, just like the above code,
```
KeySelector keySelector = config.getStatePartitioner(i, cl);
            if (keySelector != null) {
                checkState(
                        keySelector instanceof ProxyKeySelector,
                        "The state partitioner for the wrapper operator should always be ProxyKeySelector, but it is "
                                + keySelector);
                wrappedConfig.setStatePartitioner(
                        i, ((ProxyKeySelector) keySelector).getWrappedKeySelector());
            }
```